backend/elasticsearch-aws/pom.xml

```diff
@@ -17,7 +17,7 @@
     <artifactId>hibernate-search-backend-elasticsearch-aws</artifactId>
 
     <name>Hibernate Search Backend - Elasticsearch - AWS integration</name>
-    <description>Configurer for the Hibernate Search Elasticsearch backend enabling connection to an AWS-hosted Elasticsearch cluster</description>
+    <description>Configurer for the Hibernate Search Elasticsearch backend enabling connection to an AWS-hosted Elasticsearch/OpenSearch cluster</description>
```

Comment:
not related to this PR 😃 
It makes sense to mention that this lib works with ES OS in its description, but I  was wondering ... we have a lot of mentions of ES or ES cluster throughout the docs, while it is kinda similar in that it is either ES or OS cluster ... Should we somehow "combine" the two into a Search Services cluster or something like that, or... or leave it as it is 😄

Reply:
Yeah I don't have a proper solution for that... "Search Services" is too generic and IMO confusing.

The only actual solutions involving specific language would require either to use "Elasticsearch/OpenSearch" everywhere, or to simply duplicate the backend and documentation. Neither seems particularly better than the status quo...